blog/2020-10-08-FacilityLocationProblemQuickstart.adoc

```diff
@@ -0,0 +1,98 @@
+= New quickstart for the facility location problem
```

Comment:
That title is accurate. I wonder if we can make it a bit more attractive to click on to read this article. Some idea's:
"Solve the facility location problem"
"How to optimize facility location decisions"
"Get started on optimizing a facility location problem"

Reply:
I choose "Solve the facility location problem". It's simple and I think it has the power to attract readers.